recipes/xraylib/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "3.1.0" %}
+
+package:
+  name: xraylib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tschoonj/xraylib
+  git_rev: xraylib-{{ version }}
```

Comment:
Hi @ericdill, I think the preference is to use the github tarball to prevent wasting time on the CI servers by downloading the whole repo.